### DISCUSSION TITLE
How to external access, not 127.0.0.1

### DISCUSSION THREAD

**Discussion by krattika1986** [filebrowser/filebrowser#1365] [Q&A] [answered]
Sorry guys, i m lame. I installed to simple ubuntu linux. From 127.0.0.1 works well. The PC have public IP adress, firewall disabled, but web interface dont open from external PC. Please help how.

**Comment by ramiresviana** [comment, marked as answer]
By default Filebrowser will listen on the address 127.0.0.1, making it available only to the local machine. To make it accessible from the network, you will need use the `--address` parameter.

**Reply by PlusDesignSD** [reply]
hi, when a try to set the parameter to listen to another address, the response is "timeout"
installed with homebrew.
Thank you the help

**Reply by Nerahikada** [reply]
filebrowser must be stopped before any changes can be made to the settings.
